core/src/main/scripts/importer/validateData.py

```diff
@@ -965,6 +972,12 @@ def checkHeader(self, cols):
                 'that the Uniprot canonical isoform is used when drawing Pfam '
                 'domains in the mutations view',
                 extra={'line_number': self.line_number})
+        elif not 'swissprot_identifier' in self.meta_dict:
+            self.logger.warning(
+                "A SWISSPROT column was found in a file without an "
+                "associated 'swissprot_identifier' metadatum, assuming "
+                "'swissprot_identifier: name'.",
```

Comment:
:information_source:  discussing with @jjgao and @n1zea144 if we still want the default to be *name* instead of *accession* now that they have also upgraded their internal VEP installation.